<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionHistory</class>
 <widget class="QDialog" name="TransactionHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>form.title.transactionhistory</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;gui.transactionhistory.title&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="filterGroupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>561</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="title">
    <string>gui.transactionhistory.box.filter</string>
   </property>
   <widget class="QDateEdit" name="periodTo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="period1Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:400;&quot;&gt;gui.transactionhistory.label.periodfrom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="period2Label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:400;&quot;&gt;gui.transactionhistory.label.periodto&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="periodFrom">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="operationComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>gui.transactionhistory.combobox.all</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gui.transactionhistory.combobox.consume</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gui.transactionhistory.combobox.income</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="operationLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:400;&quot;&gt;gui.transactionhistory.label.operationtype&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>gui.transactionhistory.button.search</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="transactionsTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>561</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>95</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>35</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>gui.transactionhistory.column.date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>gui.transactionhistory.column.time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>gui.transactionhistory.column.amount</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>gui.transactionhistory.column.description</string>
    </property>
   </column>
  </widget>
 </widget>
 <tabstops>
  <tabstop>periodFrom</tabstop>
  <tabstop>periodTo</tabstop>
  <tabstop>operationComboBox</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>transactionsTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
